app/controllers/admin/users_controller.rb

```diff
@@ -1,28 +1,39 @@
 # frozen_string_literal: true
 
 class Admin::UsersController < Admin::BaseController
-  respond_to :html, :js
   before_action :another_user_access, only: %i[edit update destroy]
 
   def index
-    @users = User.not_portus.order(:username).page(params[:page])
     @admin_count = User.admins.count
-  end
-
-  def new
-    @user = User.new
+    @users = User.not_portus.order(:username)
+    @users_serialized = API::Entities::Users.represent(
+      @users,
+      current_user: current_user,
+      type:         :internal
+    ).to_json
   end
 
   def create
```

Comment:
The behavior is different atm and I didn't want to extend it on this PR. We need to make it accept password and if no password is sent, a token is sent back to the person who created the user. I didn't want to do this in this PR since this was already too long and kinda out of this scope. But eventually this code will go away once we change the api.

Reply:
Gotcha. Just saying because currently there's lots of duplication here and there when it comes to controllers vs API, and I'd prefer to use the API whenever possible and reduce controllers to their bare minimum :nerd_face:

Reply:
Yeah, I'm always favoring API whenever possible but user is a peculiar one that we need to be careful when getting over it.